beetsplug/web/__init__.py

```diff
@@ -69,10 +69,14 @@ def json_generator(items, root):
             first = False
         else:
             yield ','
-        yield json.dumps(_rep(item))
+        yield json.dumps(_rep(item, expand=expand))
     yield ']}'
 
 
+def is_expand(args):
```

Comment:
Do we really need this method? It might be simpler to just do this:

```python
return flask.jsonify(_rep(entities[0], expand='expand' in flask.request.args))
```

Or even work out whether or not we should expand at the top of the `resource` function and use that instead?